libs/blocks/marketo/marketo.js

```diff
@@ -166,12 +186,12 @@ export default function init(el) {
     return;
   }
 
-  if (formData['form.success.content']) {
+  if (!formData[SUCCESS_TYPE] || formData[SUCCESS_TYPE] === 'redirect') {
```

Comment:
```suggestion
  if (formData?.[SUCCESS_TYPE] === 'redirect') {
```

Reply:
Some authors do not set a success type so we want to maintain the redirect default, other types are "message" or "section".
So if there is no type or if the type is redirect then we want to check that the success content is a URL.

Reply:
This may have more clarity:
```suggestion
  formData[SUCCESS_TYPE] = formData[SUCCESS_TYPE] || 'redirect';
  if (formData[SUCCESS_TYPE] === 'redirect') {
```